cmd/juju/commands/main_test.go

```diff
@@ -236,6 +231,7 @@ var commandNames = []string{
 	"expose",
 	"generate-config", // alias for init
 	"get-config",
+	"get-configs",
```

Comment:
You may disagree, but i still thing config is a collective noun like fish or sheep.
You wouldn't say "Look at those sheeps". Similarly, get-configs seems quite wrong.